projects.js

```diff
@@ -0,0 +1,165 @@
+const projects = [
+  {
+    title: 'Multi-Post Stories',
+    company: 'CANOPY',
+    position: 'Back End Dev',
```

Comment:
- [ ] Please check the popup modal location on screens, currently, it is located not in the center:

![image](https://user-images.githubusercontent.com/77038610/151208213-a6b5488a-ed8b-4786-b9ae-c362e84ea4e2.png)
As you can see there is more space remaining on the right side, please locate it in the center.